packages/e2e-tests/plugins/plugins-api/sidebar.js

```diff
@@ -86,25 +85,22 @@
 				PluginSidebar,
 				{
 					name: 'title-sidebar',
-					title: __( 'Sidebar title plugin' )
+					title: __( 'Plugin sidebar title' ),
 				},
-				el(
-					SidebarContentsWithDataHandling,
-					{}
-				)
+				el( SidebarContentsWithDataHandling, {} )
 			),
 			el(
 				PluginSidebarMoreMenuItem,
 				{
-					target: 'title-sidebar'
+					target: 'title-sidebar',
 				},
-				__( 'Sidebar title plugin' )
+				__( 'Plugin sidebar more menu title' )
```

Comment:
I changed the name to ensure that we use the version explicitly registered by the plugin, rather than the one automatically injected.